check-environment-and-install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+return_code=0
+if [[ -n "${BROWSER_DRIVER_INSTALLER_CHROME_VERSION:-}" && -n "${BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH:-}" ]]; then
+    npx browserDriverInstaller --browser-name Chrome                                                \
+                    --browser-version "$BROWSER_DRIVER_INSTALLER_CHROME_VERSION"                    \
+                    --target-path "$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH" || return_code=$?
```

Comment:
rather than || - either use an exit trap or (set -e) as we do in our shell scripts

Reply:
We want the execution to continue and finish the other installation if one of the installations fail, no?

Reply:
I think not

Reply:
Fixed